pkg/config/node.go

```diff
@@ -19,6 +19,13 @@ type Node struct {
 	NodeIP string `json:"nodeIP"`
 }
 
+type MultiNode struct {
+	Enabled bool `json:"enabled"`
+	// only one master node is supported, no control plane HA
+	Master  string   `json:"master"`
+	Workers []string `json:"workers"`
```

Comment:
What populates the list of workers? Does the user need to know them in advance?

Reply:
I should probably remove it in this commit. I was thinking to use it for generating kubelet kubeconfig for each extra worker node. It could be handled in a follow up PR if needed when adding multi worker nodes support in microshift main binary.

Reply:
removed `Workers` and move the `Multinode` structure to a separate file pkg/config/multinode.go